src/bot/announcer.py

```
@@ -0,0 +1,216 @@
+                        successful_announcements += 1
+                except Exception as e:
+                    logger.error(f"Failed to announce in guild {guild.id}: {e}")
+
+            logger.info(f"Match announced in {successful_announcements} guilds")
+            return successful_announcements > 0
+
+        except Exception as e:
+            logger.error(f"Error announcing match: {e}", exc_info=True)
+            return False
+
+    async def send_custom_announcement(self, title: str, message: str, color: discord.Color = discord.Color.blue()):
```

Ruff (0.8.2)

Do not perform function call `discord.Color.blue` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)